### PR TITLE
adds vagrant-libvirt templates for fedora22

### PR DESCRIPTION
**Pull request by witjoh**

### CODE REVIEW

**Comment by shrug** [comment]
ping @heathseals or @shermdog

**Comment by heathseals** [comment]
i'm in the process of retooling our builders to better support building for libvirt, but i'll go ahead and run this through an ad hoc builder in the meantime.

File: templates/fedora-22/i386.libvirt.base.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    },
+
+  "builders": [
+    {
+      "name": "{{user `template_name`}}-{{user `provisioner`}}",
+      "type": "qemu",
+      "accelerator": "kvm",
+      "format": "qcow2",
+      "net_device": "virtio-net-pci",
+      "disk_interface": "virtio-scsi",
```
Comment (heathseals) [inline templates/fedora-22/i386.libvirt.base.json]: this template is missing `"headless": true,`

**Comment by heathseals** [comment]
@witjoh other than the comment above, this looks good to me. 👍

**Comment by witjoh** [comment]
@heathseals : corrected

**Pull request merged**